### PR TITLE
Apply renovate bot

### PR DESCRIPTION
**Pull request by samsonasik**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Like PR https://github.com/laminas/laminas-view/pull/137 by @gsteel 

- [x] Apply renovate
- [x] Remove dependabot config

### CODE REVIEW

**Comment by samsonasik** [comment]
@Ocramius is renovate will update composer.json for pinned rector/rector version?

```
"rector/rector": "0.12.15"
```

**Comment by Ocramius** [comment]
> will update composer.json for pinned rector/rector version?

Yes, it should be able to update pinned versions regularly, but I don't think we configured that, so in theory no action happening here.

**Comment by internalsystemerror** [comment]
@samsonasik Yes, it should be doing that by default. We set it to replace, so it will check the constraint (`~`, `^` etc), and if the new version falls outside the constraint, it will (except for majors) attempt to automerge an update. If CI fails for any reason, a PR will be opened. New majors will open a PR either way.

See: https://docs.renovatebot.com/configuration-options/#rangestrategy for how this works. We used the value `replace`.

**Comment by samsonasik** [comment]
@internalsystemerror thank you, so it is merge-able yes?

File: composer.json [hunk 1]
```diff
@@ -34,7 +34,10 @@
         }
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4"
```
Comment (internalsystemerror) [inline composer.json:39]: I believe this needs to have the patch version too, unfortunately. I'm not at my laptop to test right now, but I thought that gave me an error when running `composer install|update`. Does it work for you?

**Review by internalsystemerror — APPROVED** [review]
If using just `"7.4"` works for `config.platform.php` then LGTM.

File: composer.json (hunk 1 above)
Comment (samsonasik) [reply composer.json:39]: Yes, it works, CI also run composer install for verify in php 7.4, 8.0, and 8.1

**Comment by samsonasik** [comment]
I am merging it, thank you @Ocramius @internalsystemerror for the review.

File: composer.json (hunk 1 above)
Comment (internalsystemerror) [reply composer.json:39]: I think mwop updated the CI so that it removes this key before running though, that's why I wanted to check.

Comment (samsonasik) [reply composer.json:39]: @internalsystemerror ok, just let me know if I can do something, thank you.

**Pull request merged**